extensions/default/src/commandsModule.ts

```diff
@@ -432,14 +432,16 @@ const commandsModule = ({
         // This findOrCreateViewport returns either one of the updatedViewports
         // returned from the HP service OR if there is not one from the HP service then
         // simply returns what was in the previous state.
-        const findOrCreateViewport = (viewportIndex: number) => {
+        const findOrCreateViewport = (position: number) => {
           const viewport = updatedViewports.find(
-            viewport => viewport.viewportIndex === viewportIndex
+            viewport => viewport.pos === position
```

Comment:
I am a bit concerned that when double clicking to one-up while in MPR and then swapping in a new (reconstructable) series, the user is left stuck in one-up until exiting MPR (i.e. they cannot double click again to return to the 3x1 MPR view). I understand that this PR was NOT responsible for that, but the changes here brought that to light.

I am still concerned that there is no `viewport.pos` defined for this code to ever work.